src/ptyfwd/ptyfwd.c

```diff
@@ -0,0 +1,190 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <getopt.h>
+#include <unistd.h>
+
+#include "alloc-util.h"
+#include "build.h"
+#include "main-func.h"
+#include "pretty-print.h"
+#include "ptyfwd-util.h"
+#include "fd-util.h"
+#include "event-util.h"
+#include "strv.h"
+
+static bool arg_quiet = false;
+static char *arg_background = NULL;
+
+STATIC_DESTRUCTOR_REGISTER(arg_background, freep);
+
+static int help(int argc, char *argv[], void *userdata) {
+        _cleanup_free_ char *link = NULL;
+        int r;
+
+        r = terminal_urlify_man("systemd-pty-forward", "1", &link);
+        if (r < 0)
+                return log_oom();
+
+        printf("%1$s  [OPTIONS...] COMMAND ...\n"
+               "\n%5$sRun command with a custom terminal background color and title.%6$s\n"
+               "\n%3$sOptions:%4$s\n"
+               "  -h --help              Show this help\n"
+               "     --version           Print version\n"
+               "  -q --quiet             Suppress information messages during runtime\n"
+               "     --background=COLOR  Set ANSI color for background\n"
```

Comment:
With `run0` one could easily use `SYSTEMD_TINT_BACKGROUND=0` to disable tinting as it had a default background. With this tool people will always have to pass a background color thus `SYSTEMD_TINT_BACKGROUND` would not work. It would be great if there were still a way to disable the tinting in this case